utils/date.ts

```diff
@@ -0,0 +1,22 @@
+import { format } from "date-fns";
+
+const date = new Date();
+
+export const CURRENT_YEAR = date.getFullYear();
+
+/*
+  - formats date correctly, regardless of timezone
+  - used on EventsPage and PressPage layouts
+*/
+export const getParsedDate = (date: Date, dateFormat: string) => {
+  const adjustedDate = new Date(
+    // getTime() returns number of milliseconds between 1 January 1970 00:00:00 UTC and the given date
+    date.getTime() +
+      // getTimeZoneOffset returns the time zone difference, in minutes, from current locale to UTC
+      // multiplying by '60' gives us the difference in seconds
+      // multiplying by '1000' gives us the difference in milliseconds
+      date.getTimezoneOffset() * 60 * 1000
+  );
+
+  return format(adjustedDate, dateFormat);
+};
```

Comment:
I don't know enough about timezones to comment on this. Does `date-fns` have anything built in?

Reply:
Date-fns doesn't come with any built in timezone utilities.